src/main/java/apaintus/models/Attribute.java

```diff
@@ -1,5 +1,29 @@
 package apaintus.models;
 
 public enum Attribute {
-    FILL_COLOR, STROKE_COLOR, STROKE_SIZE, COORDINATE_X, COORDINATE_Y, WIDTH, HEIGHT, ORIENTATION, TEXT,
+    COORDINATE_X,
```

Comment:
Why only add the toString() function to a selected few enums, for consistency every one of them should have a toString(). Just to ensure that we have the same convention for all Attributes, because right now, they don't.